src/ConsoleBuffer/Commands/ControlSequence.cs

```diff
@@ -2,9 +2,26 @@ namespace ConsoleBuffer.Commands
 {
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics;
 
     public abstract class ControlSequence : Base
     {
+        public class ParsedParameters
```

Comment:
In general prefer sealed classes unless you expect inheritance.

Reply:
Done.